### DISCUSSION TITLE
ST7793 240x400 support

### DISCUSSION THREAD

**Discussion by sude22** [Bodmer/TFT_eSPI#3071] [New processor/display support or capability request]
Hello, and first of all many thanks for your work on this great software. I have already used it for several projects and I am delighted with the smooth fonts.
Now I have a new [display](https://www.aliexpress.com/item/1005005617019898.html?spm=a2g0o.order_list.order_list_main.22.15481802mZUvHg) for another project with a ESP32 D1 mini.

This driver seems to be not supported right now. 
Could you please give me a short hint where to start?

**Comment by sude22** [comment]
As further information, I would like to share my current status with the new display.
The hardware setup works, I was able to run a demo with the "MCUFRIEND_kbv" library. 
In a post I found the information that the drivers for the chips SPFD5420, ST7793, ILI9326, R61509V should be very similar or the same.
If I can provide other information or do tests please let me know.

**Comment by Avtollic** [comment]
доброго часу! будь ласка допоможіть підключити запустити st7793 на esp8266. завчасно дякую!!!
